R/iterators.R

```diff
@@ -1144,14 +1144,11 @@ simple_es_index <- function(x, i, na_ok = FALSE) {
 #' @name igraph-vs-attributes
 #' @export
 `[[<-.igraph.vs` <- function(x, i, value) {
-  if (
-    !"name" %in% names(attributes(value)) ||
-      !"value" %in% names(attributes(value))
```

Comment:
I don't understand how value could be missing. I've added tests for when name is missing, but value missing would mean nothing on the right side of the arrow?! Am I missing something obvious?

In any case we need to improve the mysterious "Invalid indexing" error.